src/lib/components/footer.svelte

```diff
@@ -1,42 +1,44 @@
 <div class="container">
   <div>
     <div>
+      <p>Ophavsret &#169; 2021,</p>
+      <p>Vermouth NU.</p>
+      <p>CVR - 40649395</p>
+    </div>
+  </div>
+  <div>
+    <div class="right">
       <div class="social-media">
-        <h3>F&oslash;lg os</h3>
+        <p>F&oslash;lg os</p>
         <a href="https://www.instagram.com">
-          <h3>Ig</h3>
+          <p>Ig</p>
         </a>
 
         <a href="https://es-es.facebook.com">
-          <h3>Fb</h3>
+          <p>Fb</p>
         </a>
       </div>
-      <h3>Ophavsret &#169; 2021,</h3>
-      <h3>Vermouth NU.</h3>
-      <h3>Drevet af Shopify</h3>
-    </div>
-  </div>
-  <div>
-    <div class="right">
-      <h3>Newsletter</h3>
-      <h3>tlf:784-449-333</h3>
-      <h3>noseymeil@gmail.com</h3>
-      <h3>Nyhedsbrev</h3>
+      <a href="mailto:info@vermouth.nu">
+        <p>info@vermouth.nu</p>
+      </a>
+      <p>Nyhedsbrev</p>
     </div>
   </div>
 </div>
 
 <style>
-  a h3 {
+  .social-media a p {
```

Comment:
link component should have ```color: inherit``` already, why is this needed ?

Reply:
I don't know what you mean, the links of the social networks are in red, that's why that style